src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/SelectAttributeType.php

```diff
@@ -95,7 +95,7 @@ public function configureOptions(OptionsResolver $resolver): void
                             continue;
                         }
 
-                        $choices[$key] = $choice[$this->defaultLocaleCode];
+                        $choices[$key] = $choice[$this->defaultLocaleCode] ?? '';
```

Comment:
As I came here once again. I wanted to ask one more thing. If I understood correctly, the choice will be just empty in this case. What will happen with more than one undefined choice? Wouldn't it be better to just skip values like this? What kind of behavior would you expect to have here?

Reply:
In first time, I want only solve the problem.
It's true, the value need to be skipped. But why, I will be change my fix and the test, if check never pass ?

Reply:
I would say, that removing the empty string from https://github.com/Sylius/Sylius/pull/11150/files#diff-64baa708ca6b415da2aa24111f3655dcR30 would be enough.

Reply:
@lchrusciel I have change the behavior. It's OK ?